### PR TITLE
Add Unblinking Flame Aura, Unblinking Flame Emblem, Unblinking Flame Ignition focus spells

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: packs/data/spell-effects.db/spell-effect-unblinking-flame-emblem.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 3
+        },
+        "level": {
+            "value": 7
+        },
+        "rules": [],
```
Comment (TikaelSol) [inline packs/data/spell-effects.db/spell-effect-unblinking-flame-emblem.json:19]: Why have an effect if it isn't carrying the rule info?

File: packs/data/spell-effects.db/spell-effect-unblinking-flame-ignition.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                "selector": "seeInvisibility"
+            },
+            {
+                "key": "BaseSpeed",
+                "selector": "fly",
+                "value": "@actor.attributes.speed.value"
+            }
+        ],
+        "source": {
+            "value": ""
```
Comment (TikaelSol) [inline packs/data/spell-effects.db/spell-effect-unblinking-flame-ignition.json:41]:
```suggestion
            "value": "Pathfinder Lost Omens: Impossible Lands"
```

File: packs/data/spell-effects.db/spell-effect-unblinking-flame-emblem.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        "source": {
+            "value": ""
```
Comment (TikaelSol) [inline packs/data/spell-effects.db/spell-effect-unblinking-flame-emblem.json:21]:
```suggestion
            "value": "Pathfinder Lost Omens: Impossible Lands"
```

File: packs/data/spell-effects.db/spell-effect-unblinking-flame-aura.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                "value": 2
+            },
+            {
+                "key": "Resistance",
+                "type": "cold",
+                "value": "2*@item.level"
+            }
+        ],
+        "source": {
+            "value": ""
```
Comment (TikaelSol) [inline packs/data/spell-effects.db/spell-effect-unblinking-flame-aura.json:33]:
```suggestion
            "value": "Pathfinder Lost Omens: Impossible Lands"
```

File: packs/data/spell-effects.db/spell-effect-unblinking-flame-emblem.json (hunk 1 above)
Comment (simonward) [reply packs/data/spell-effects.db/spell-effect-unblinking-flame-emblem.json:19]: To mark the target, and as a placeholder to contain the rule info once it's supported. I can remove it if you prefer.

Comment (simonward) [reply packs/data/spell-effects.db/spell-effect-unblinking-flame-emblem.json]: Removed the effect for now.

**Pull request merged**